lib/src/serializable.dart

```diff
@@ -26,10 +25,16 @@ import 'package:w_module/src/event.dart';
 import 'package:w_module/src/module.dart';
 
 // Any classes / methods that are going to be reflected must annotate with this
-class Reflectable {
-  const Reflectable();
+class Reflectable extends reflect.Reflectable {
```

Comment:
What's the export for this file look like, this won't get exported, will it?

Reply:
It is exported intentionally. I made no changes to the public api here, just changed the `extends` here to use the reflectable package. 
FYI The `@Reflectable` annotation is used in w_viewer.

Reply:
Oh OK, gotcha. Why the `_reflector` instance?

Reply:
the _reflector instance is used further down to reflect on things. That's how you use it according to the docs https://github.com/dart-lang/reflectable/tree/master/reflectable#usage

Reply:
Is this still going to work then?

https://github.com/Workiva/w_viewer/blob/00e7ce49d522123eea0bc1b77bb6182b19f86441/lib/src/modules/content_renderer/content_renderer_api.dart#L22